Library/Homebrew/unpack_strategy/zip.rb

```diff
@@ -27,10 +27,17 @@ def self.can_extract?(path)
               .returns(SystemCommand::Result)
     }
     def extract_to_dir(unpack_dir, basename:, verbose:)
+      unzip = begin
+        Formula["unzip"]
+      rescue FormulaUnavailableError
+        nil
```

Comment:
What happens if there's no system `unzip` and this is `nil`?

Reply:
`PATH.new` ignores nil so it'll just use `ENV["PATH"]`.

Reply:
(https://github.com/Homebrew/brew/blob/f07a44f5079009524002eeb8bdf5fc1c6ae2b55a/Library/Homebrew/PATH.rb#L92)

Reply:
Oh misread your question. The command will fail to execute if there's no system unzip. But this really should only be nil if homebrew-core is not installed, and I imagine there are many other issues in that case. All the other unpack strategies will actually error since they don't have a `FormulaUnavailableError` check (we do here because it's needed for the rspec tests).

Reply:
Gotcha, that all seems fine to me then 👍🏻